OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -143,6 +143,7 @@ public void onCreate(Bundle savedInstanceState) {
         Map<String, String> values = new HashMap<>();
         values.put(Analytics.Keys.TOPIC_ID, discussionThread.getTopicId());
         values.put(Analytics.Keys.THREAD_ID, discussionThread.getIdentifier());
+        values.put(Analytics.Keys.AUTHOR, discussionThread.getAuthor());
```

Comment:
Thanks for pointing that out. That behavior would actually be inconsistent with what the iOS app will do, so I'm going to change this to leave out the author field if the post is anonymous.